packages/app/android/app/build.gradle

```diff
@@ -45,15 +51,22 @@ android {
 
     defaultConfig {
         applicationId "com.p2panda.meli"
-        // You can update the following values to match your application needs.
-        // For more information, see: https://docs.flutter.dev/deployment/android#reviewing-the-gradle-build-configuration.
         minSdkVersion 21
```

Comment:
I've tested the app with SDK versions 21, 22, 23, 24, and 25 and we're compatible from 23 on. If I change this value to 23 though the .apk size doubles 😱

Reply:
Should we keep it at 21 for the apk size, even though the app breaks in these versions and hope that noone has such an old Android?

Reply:
> If I change this value to 23 though the .apk size doubles 😱 Why?

That's super strange....

Reply:
https://stackoverflow.com/questions/62440105/apk-size-increased-35mb-when-bumping-min-sdk-from-21-to-24/63417612#63417612